site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -269,29 +269,29 @@ include::_tenant-generator-length-config-rules.adoc[]
 :generator_display_name!:
 +
 
-[field]#tenant.externalIdentifierConfiguration.twoFactorOneTimeCodeIdGenerator.type# [type]#[String]# [required]#Required# [default]#defaults to `randomAlphaNumeric`# [since]#Available since 1.27.0#::
+[field]#tenant.externalIdentifierConfiguration.twoFactorOneTimeCodeIdGenerator.type# [type]#[String]# [optional]#Optional# [default]#defaults to `randomAlphaNumeric`# [since]#Available since 1.27.0#::
 +
```

Comment:
If I'm looking at the correct place in the code [ExternalIdentifierConfiguration Line 67](https://github.com/FusionAuth/fusionauth-java-client/blob/1.30.0/src/main/java/io/fusionauth/domain/ExternalIdentifierConfiguration.java#L67) shouldn't this default to `randomDigits`?

Reply:
You are correct sir. Good catch.